File: app/src/main/java/ru/vincetti/vimoney/data/models/json/ConfigFile.java
package ru.vincetti.vimoney.data.models.json;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ConfigFile {

    @SerializedName("accounts")
    private List<AccountsItem> accounts;

    @SerializedName("user")
    private User user;

    @SerializedName("date_edit")
    private long date_edit;

    @SerializedName("currency")
    private List<CurrencyItem> currency;

    @SerializedName("transactions")
    private List<TransactionsItem> transactions;

    public List<AccountsItem> getAccounts() {
        return accounts;
    }

    public User getUser() {
        return user;
    }

    public long getDateEdit() {
        return date_edit;
    }

    public List<CurrencyItem> getCurrency(){
        return currency;
    }

    public List<TransactionsItem> getTransactions(){
        return transactions;
    }
}

File: app/src/main/java/ru/vincetti/vimoney/data/models/json/ConfigFile.kt
package ru.vincetti.vimoney.data.models.json

import com.google.gson.annotations.SerializedName

data class ConfigFile(

        @SerializedName("accounts")
        val accounts: List<AccountsItem>,

        @SerializedName("user")
        val user: User,

        @SerializedName("date_edit")
        val date_edit: Long,

        @SerializedName("currency")
        val currency: List<CurrencyItem>,

        @SerializedName("transactions")
        val transactions: List<TransactionsItem>,

        @SerializedName("categories")
        val categories: List<CategoriesItem>
)
